### ISSUE TITLE
Wbepack build [hash] substitution is not resolved in public path

### ISSUE THREAD

**Issue by martinkadlec0** [kisenka/svg-sprite-loader#295]
**Do you want to request a feature, report a bug or ask a question?**
Bug/Feature ?

**What is the current behavior?**
When I set publicPath to `app/[hash]` it is resolved to `app/[hash]`.

**What is the expected behavior?**
When I set publicPath to `app/[hash]` it is resolved to `app/buildhash`.

**Please tell us about your environment:**

- Node.js version: Node 10.4
- webpack version: 4.12
- svg-sprite-loader version: 3.8.0
- OS type & version: macOS High Sierra

**Comment by kisenka** [comment]
Could you please create repo with minimal setup to demonstrate a problem (package.json, webpack config, SVG image and piece of your code). If you don't want to create a repository - you can create a [gist](http://gist.github.com) with multiple files.

**Comment by martinkadlec0** [comment]
@kisenka added link to repo

**Comment by kisenka** [comment]
@martinkadlec0 I suggest not to use `[hash]` interpolation in `publicPath` loader config option, because it's not safe to replace this token in whole CSS. I remove publicPath param from loader config and it's worked:

```css
.test {
    background: url(http://cdn.com/644577553952bb274471/sprite.svg#rocket-usage);
}
```

**Comment by martinkadlec0** [comment]
@kisenka no, notice that the public path for the sprite is different to the webpack public path.

I actually need this to set publicPath of sprite to my instead of cdn domain.

**Comment by kisenka** [comment]
You talking about hash of compilation, right?

**Comment by martinkadlec0** [comment]
Yes
